### DISCUSSION TITLE
[Feature]: Ability for selecting own source of Node installation binaries

### DISCUSSION THREAD

**Discussion by tomaszwojciechowski** [coreybutler/nvm-windows#989] [Ideas & Feature Requests]
### Description of the new feature / enhancement

I would to be able to configure my own source of Node installation binaries.

### Scenario when this would be used?

E.g. in situation where there is limitation of addresses from which Node could be downloaded.

**Comment by coreybutler** [comment]
This is something on our roadmap for [Runtime](https://github.com/coreybutler/nvm-windows/wiki/Runtime). However; there are quite a few questions about what kind of sources should be supported, so I'm moving this to a discussion.

**Reply by coreybutler** [reply]
@tomaszwojciechowski - this subject has come up before. There are a few sources that have been mentioned before:

1. Self-Hosted Mirror (i.e. replicates nodejs.org)
2. Alternative mirrors (Taobao, Nodesource, etc)
3. Local Directory/Network Directory (i.e. behind the firewall)

Most people who have requested this do _not_ have the infrastructure or know-how to host their own Node mirror (there are exceptions). Others do not have internet access or have sporadic access.

Ultimately, I'd like to know more about how people plan to source both Node and npm/yarn/etc.
